src/test/functional/features/pay-and-submit.feature

```diff
@@ -0,0 +1,29 @@
+# TODO uncomment when joint application can be submitted
```

Comment:
When will this be re-added?

Reply:
i guess when the full app2 journey is implemented as i cannot get it in state. (tho i think Lara is working on it).